partcad/tests/unit/test_test.py

```
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
```

Ruff (0.8.2)

Shebang is present but file is not executable

(EXE001)